core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.ts

```diff
@@ -33,6 +33,10 @@ angular.module('oppia').directive('sideNavigationBar', [
         };
         ctrl.$onInit = function() {
           ctrl.currentUrl = $window.location.pathname;
+          ctrl.showMe = false;
+        };
+        ctrl.myFunc = function() {
```

Comment:
Please use a more descriptive name for the function. "myFunc" seems a bit unclear about what the function actually does.

Reply:
Done!